### ISSUE TITLE
[User Accounts] Add ability to batch edit status of users

### ISSUE THREAD

**Issue by johnsaigle** [aces/Loris#4022]
@LeighMac has requested the ability to (de)activate users on mass from the user_accounts module. It would be useful to be able to apply any of the user accounts functionality to a group of users, such as generating new passwords, set to no longer pending approval, etc.

**Comment by HenriRabalais** [comment]
this needs the `rowActions` mechanism from #10285 before it can be done, since that's what gives DataTable bulk actions on selected rows.

it doesn't resolve this on its own though, this will need its own follow-up PR stacked on top once it lands. #10285 is itself stacked on #10252, so that has to merge first.
